__tests__/app.test.js

```diff
@@ -94,6 +94,25 @@ describe("/api/articles", () => {
     })
 })
 
+describe("/api/users", () => {
+    describe("GET", () => {
+        test("GET 200: Responds with an array of all user objects", () => {
+            return request(app)
+            .get("/api/users")
+            .expect(200)
+            .then(({body}) => {
```

Comment:
You should also test for the length of the array to ensure the forEach below doesn't give you a false positive